modules/nodes-sigstore-using-about.adoc

```diff
@@ -0,0 +1,9 @@
+// Module included in the following assemblies:
+//
+// * nodes/nodes-sigstore-using.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="nodes-sigstore-using-about_{context}"]
+= About the node metrics dashboard
+
+The Sigstore project enables developers to sign-off on what they build and administrators to verify signatures and monitor workflows at scale. With the SigStore project, signatures can be stored in the same registry as the build images. A second server is not needed. The identity piece of a signature is tied to the OpenID Connect (OIDC) identity through the Fulcio certificate authority, which simplifies the signature process by allowing key-less signing. Additionally, SigStore includes Rekor, which records signature metadata to an immutable, tamper-resistant ledger.
```

Comment:
I think it's just `sigstore`

```suggestion
The sigstore project enables developers to sign-off on what they build and administrators to verify signatures and monitor workflows at scale. With the sigstore project, signatures can be stored in the same registry as the build images. A second server is not needed. The identity piece of a signature is tied to the OpenID Connect (OIDC) identity through the Fulcio certificate authority, which simplifies the signature process by allowing key-less signing. Additionally, sigstore includes Rekor, which records signature metadata to an immutable, tamper-resistant ledger.
```

Reply:
Oops! I think I was trying to camel-case it for some reason. I'll fix that. Thanks!